tracee-rules/benchmark/signature/golang/code_injection.go

```diff
@@ -28,15 +34,8 @@ func NewCodeInjectionSignature() (types.Signature, error) {
 	}, nil
 }
 
-var processMemFileRegexp *regexp.Regexp
```

Comment:
https://golang.org/doc/articles/race_detector#Unprotected_global_variable

The `Init()` method is invoked by multiple goroutines created by benchmark runner.